roles/foreman_provisioning/tasks/configure_ubuntu_24_04.yml

```diff
@@ -0,0 +1,29 @@
+- name: "Find all relevant templates" # noqa: args[module]
+  theforeman.foreman.resource_info:
+    resource: provisioning_templates
+    search: name ~ "Preseed default" or name = "Linux host_init_config default"
+  register: result
+
+- name: "Ensure Ubuntu 24.04" # noqa: args[module]
+  theforeman.foreman.operatingsystem:
+    name: Ubuntu
+    family: Debian
+    major: "24.04"
+    release_name: artful
+    architectures:
+      - x86_64
+    media:
+      - Ubuntu mirror
+    provisioning_templates: "{{ result.resources | map(attribute='name') }}"
+    ptables:
+      - Preseed default
+    state: present
+    password_hash: "SHA256"
+
+- name: "Set default template for Ubuntu 24.04" # noqa: args[module]
+  theforeman.foreman.os_default_template:
+    operatingsystem: "Ubuntu 24.04"
+    template_kind: "{{ item.template_kind_name }}"
+    provisioning_template: "{{ item.name }}"
+    state: present
+  loop: "{{ result.resources | sort(attribute='name') | unique(attribute='template_kind_name') }}"
```

Comment:
I think Ubuntu 24 needs the "Autoinstall" type templates, not the classic ones.

Reply:
Is regular debian fine with the non-autoinstall ones?

Reply:
Yeah, it's just ubuntu who switched to the new installer.

Reply:
Well, let's just hope there won't be any other flavor of the preseed templates.